keep-ui/app/incidents/incident.tsx

```diff
@@ -104,24 +128,24 @@ export default function Incident() {
                 </Button>
               </div>
             </div>
+
             <Card className="mt-10 flex-grow">
```

Comment:
Remove margin in favor of gap on parent 
```suggestion
            <Card className="flex-grow">
```